golem_messages/validators.py

```diff
@@ -1,7 +1,7 @@
 from golem_messages import exceptions
 
 
-def validate_varchar(field_name, value, max_length):
+def validate_varchar(field_name, value, max_length=float('inf')):
```

Comment:
I think that instead of default `float('inf')`, you should check if `max_length` is not None, and then validate against it, otherwise no.

Reply:
I thought a little bit about it and I have decided to delete `float('inf')` from here.